CHANGELOG.md

```diff
@@ -38,71 +38,61 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.20.0](https://github.com/cds-snc/platform-forms-client/compare/v3.19.0...v3.20.0) (2024-08-26)
 
-
 ### Features
 
-* Updates Repeating Sets, Pages and Rules to work together on the Review page and form output ([#4186](https://github.com/cds-snc/platform-forms-client/issues/4186)) ([5422d6c](https://github.com/cds-snc/platform-forms-client/commit/5422d6c1800a5ff961270e1effc224718558926a))
```

Comment:
Any idea why this is landing in your PR

I see the diff is the `*` getting changed to `-`

Reply:
git checkout origin/develop -- CHANGELOG.md pulls the *'s changing into -'s as well. Super strange.

Reply:
Found the issue : It's the linter forcing itself onto Changelog.md again. Will fix!~